src/app/plugins/addclasschans.plugin.ts

```diff
@@ -68,6 +47,42 @@ export default class AddClassChannelsPlugin extends Plugin {
     }
   }
 
+  private _getNewChanMessage(id: string): string {
+    return 'Welcome to the class!\n\n' +
+      `**If it has not been done so already, please post the <#${id}> ` +
+      'to webcourses to have your classmates join you in this channel.**\n\n' +
+      '**For TAs**\n' +
+      'If you are a TA for this course, reach out to a Moderator to have the ' +
+      'TA role added to your user and register as the TA in this channel using ' +
+      '`!ta register`. Students in the class can ask the TA a question with a ' +
+      'pingable command `!ta ask`.\n\n' +
+      '**For Professors**\n' +
+      'If you are a professor for this course, reach out to a Moderator to have the ' +
+      'Professor role added to your user.\n\n' +
+      '**New Create Voice Chat Feature**\n' +
+      'You can now create a temporary voice channel for your class by using `!createclassvoice` ' +
+      '(or shorthand `!createvc`) in your class channel. Only people in the channel will be able to ' +
+      'access the temporary channel so you can have private study sessions without the concern of ' +
+      'randos jumping in.\n\n' +
+      '**Need Help?**\n' +
+      'In any channel, use `!help` to see what options are available from our bot, Lion. ' +
+      'Feel free to reach out to any Moderator with questions or concerns for the server.\n\n' +
+      'Have a great semester!';
+  }
+
+  private async _getInvChan() {
+    const ret = this.container.guildService
+      .get()
+      .channels.cache.find((c) => c.name === Constants.Channels.Info.ClassInvite);
+    if (!ret) {
+      return (await this.container.guildService
+        .get()
+        .channels.fetch())
+        .find((c) => c.name === Constants.Channels.Info.ClassInvite);
+    }
+    return ret;
+  };
```

Comment:
Sorry been gone all weekend so, just seeing this
but `GuildService` has a `getChannel` method that takes a string name of a channel

Reply:
True, I should use that instead. I was thinking of cache misses, but that shouldn’t happen since we have the GUILDS intent.